lib/src/functions.cpp

```diff
@@ -294,7 +294,7 @@ bool copyRecursively(QString srcFilePath, QString tgtFilePath)
 	// Try to create the target directory
 	QDir targetDir(tgtFilePath);
 	targetDir.cdUp();
-	if (!targetDir.mkdir(QDir(tgtFilePath).dirName())) {
+	if (!targetDir.mkpath(QDir(tgtFilePath).dirName())) {
```

Comment:
I just merged `master` into `develop`, so you can rebase your PR and you won't need to include this. 👌